wp-content/themes/core/components/accordion/accordion.php

```diff
@@ -0,0 +1,55 @@
+<?php
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$controller = \Tribe\Project\Templates\Components\accordion\Controller::factory( $args );
+?>
+<div
```

Comment:
Oof this is difficult to read/grok quickly. I winder if we should just pass in the additional attributes from the controller instead of mixing the static & dynamic attributes here?

Reply:
Agreed and they should be set as required attributes